docs/source/progress.rst

```diff
@@ -36,19 +36,18 @@ The summary statistics are counts of the records in your dataset.
 - Labeled records: the number of records that you labeled as relevant or irrelevant, including those you added as prior knowledge.
 - Relevant records: the number of records that you labeled as relevant, including those you added as prior knowledge.
 - Irrelevant records: the number of records that you labeled as irrelevant, including those you added as prior knowledge.
-- Irrelevant since last relevant: the number of irrelevant records you have seen since the last relevant record. The larger this number (relatively), the more likely you have seen the relevant records.
+- Irrelevant since last relevant: the number of irrelevant records you have seen since the last relevant record. 
+
 
 Charts
 ~~~~~~
 
 The charts on the analytics page can be useful to monitor your progress. There
-is a *Progress* and a *Recall* chart. You will probably see a decline in the
-number of relevant items you find. This helps to decide when to stop. The
-charts do not include prior knowledge and are most relevant after you have
-screened a couple of hundreds of records.
+is a *Progress* and a *Recall* chart. The charts do not include prior
+knowledge and are most relevant after you have screened a couple of records.
```

Comment:
```suggestion
knowledge and are most relevant after you have screened a couple of records.
```

How many records? I think this should be at least 10 to make the progress chart work.